### PR TITLE
DOC-2832: add 3rd-party package info to Development page

### PR DESCRIPTION
**Pull request by dwdougherty**
Updates to the Development page that describe how to use a third-party package with a Redis Stack function.

### CODE REVIEW

**Comment by dwdougherty** [comment]
@thomascaudron Please review and merge.

File: docs/docs/Development.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ description: >
   How to develop for triggers and functions
 ---
 
-To aid in the development of new libraries of triggers and functions you can use the type declaration files for the triggers and functions API. This can be used by your preferred development environment to provide autocompletion and type checking based on the type information. You can install it using the following command:
+To aid in the development of new libraries of triggers and functions, you can use the type declaration files for the [triggers and functions API](/docs/interact/programmability/triggers-and-functions/concepts/javascript_api/), which allows your preferred development environment to provide autocompletion and type checking. You can install this information using the following command:
 
 ```bash
 npm install https://gitpkg.now.sh/RedisGears/RedisGears/js_api
```
Comment (thomascaudron) [inline docs/docs/Development.md:12]: @MeirShpilraien should we add --save-dev so it is only saved as a devdependency?

File: docs/docs/Development.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ description: >
...
 ```
 
-Or add it directly as a dependency to your `package.json`:
+Or you can add it as a dependency to your `package.json` file:
```
Comment (thomascaudron) [inline docs/docs/Development.md:15]: We should specify devDependency, as it is only used during development and we do not want to bundle it when uploading the library.

**Comment by dwdougherty** [comment]
@thomascaudron Please re-review, approve, and merge when you are able. Thank you!

**Review by thomascaudron — APPROVED** [review]

**Pull request merged**